src/generators/genhl.ml

```diff
@@ -3329,15 +3329,20 @@ let generate_static ctx c f =
 					let gen_content() =
 						op ctx (OThrow (make_string ctx ("Requires compiling with -D hl-ver=" ^ ver ^ ".0 or higher") null_pos));
 					in
-					ignore(make_fun ctx ~gen_content (s_type_path c.cl_path,f.cf_name) (alloc_fid ctx c f) (match f.cf_expr with Some { eexpr = TFunction f } -> f | _ -> abort "Missing function body" f.cf_pos) None None)
+					ignore(match f.cf_expr with
```

Comment:
Those outer `ignore` are not needed. But since those are nested `match`, you need to wrap them in either `(match ...)` or `begin match ... end`